### ISSUE TITLE
Ability to cut/not render line at length or percentage

### ISSUE THREAD

**Issue by BHare1985** [mapnik/mapnik#3703]
Is there anyway I can take a line that is say 100 meters in length and have mapnik draw the beginning 10% or last 10%?

I saw that line-dasharray could be misused and if you knew the length of the line, you could perhaps trick is so the gap was the part you wanted to not render (10%) and the line portion was rest.

**Comment by talaj** [comment]
I don't think this is possible with line symbolizer, but if you just want to visualize where the beginning or end of a line is heading, you can use markers symbolizer with placement `vertex-first` or `vertex-last`.
